compiler/src/main/java/com/squareup/anvil/compiler/Utils.kt

```diff
@@ -140,3 +140,35 @@ internal fun File.requireDelete() {
     throw AnvilCompilationException("Could not delete file: $this")
   }
 }
+
+/**
+ * Transforms the elements of the receiver collection and adds the results to a set.
+ *
+ * @param destination The destination set where the transformed
+ *   elements are placed. By default, it is an empty mutable set.
+ * @param transform Maps elements of the receiver collection to the output set.
+ * @receiver The collection to be transformed.
+ * @return A set containing the transformed elements from the receiver collection.
+ */
+internal inline fun <C : Collection<T>, T, R> C.mapToSet(
```

Comment:
Really wish this was in the stdlib sometimes heh

Reply:
> sometimes

Yeah, pretty much every time I have to type it out manually.  Then after a handful of times I'll usually give in and add it. 😀